package.json

```diff
@@ -48,7 +48,7 @@
     "@babel/eslint-parser": "^7.17.0",
     "@babel/plugin-transform-react-jsx": "^7.12.11",
     "@babel/preset-env": "^7.16.0",
-    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-react": "^7.9.0",
```

Comment:
bumping the dev dependency alone is not enough. For our users, what matters for the package managers is the `peerDependencies` section.

btw, we should update our `features` utility to make it enforce versions based on `peerDependencies` rather than `devDependencies` now that we properly define peer dependencies. Or we should drop that runtime check entirely in favor of letting yarn/npm/pnpm show the warning at installation time.

Reply:
Oh yes, forgot about peerDeps, thanks

Reply:
> btw, we should update our `features` utility to make it enforce versions based on `peerDependencies` rather than `devDependencies` now that we properly define peer dependencies. Or we should drop that runtime check entirely in favor of letting yarn/npm/pnpm show the warning at installation time.

Totally! Maybe we can do 1) for v5, and 2) for v6?

Reply:
the first change can even be done in a minor version (as that's about changing the internal implementation only). Our peerDependencies is already what package managers will report.